ci-utils/build_conda.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+set -e
+
```

Comment:
I don't think we need this file since there is no CPP target similar to Nyxus, in this case